lib/omnibus/health_check.rb

```diff
@@ -51,6 +51,16 @@ class HealthCheck
       /libutil\.so/,
     ].freeze
 
+    WRLINUX_WHITELIST_LIBS = [
+      /libc\.so/,
+      /libcrypt\.so/,
+      /libdl\.so/,
+      /libm\.so/,
+      /libpthread\.so/,
+      /librt\.so/,
+      /libutil\.so/,
+    ].freeze
+
```

Comment:
All of these libraries exist in the default `WHITELIST_LIBS`...we shouldn't need to maintain a special list for Wind River.